WordPress/Classes/ViewRelated/Gutenberg/Layout Picker/CategorySectionTableViewCell.swift

```diff
@@ -76,23 +95,24 @@ extension LayoutPickerSectionTableViewCell: UICollectionViewDelegate {
     }
 
     func collectionView(_ collectionView: UICollectionView, didSelectItemAt indexPath: IndexPath) {
-        delegate?.didSelectLayoutAt(indexPath.item, forCell: self)
+        guard let slug = section?.categorySlug else { return }
```

Comment:
Technically, this is really an indication that the `section` is missing, as the `slug` is non-optional. I think this could be written a bit better to make that more clear.. maybe `guard let section = section else { return }` and dereference on the next line would be more clear. :thinking:

Reply:
No this is fine. My confusion really started with the delegate's signature but I don't have a strong feeling here.